site/en/tutorials/eager/index.md

```diff
@@ -5,8 +5,9 @@ operations. Write custom layers, forward passes, and training loops with
 auto&nbsp;differentiation. Start with these notebooks, then read the
 [eager execution guide](../../guide/eager).
 
-1. <span>[Eager execution](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/eager/python/examples/notebooks/eager_basics.ipynb){:.external}</span>
-2. <span>[Automatic differentiation and gradient tape](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/eager/python/examples/notebooks/automatic_differentiation.ipynb){:.external}</span>
-3. <span>[Custom training: basics](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/eager/python/examples/notebooks/custom_training.ipynb){:.external}</span>
-4. <span>[Custom layers](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/eager/python/examples/notebooks/custom_layers.ipynb){:.external}</span>
-5. [Custom training: walkthrough](/tutorials/eager/custom_training_walkthrough)
+1. [Eager execution](eager_basics.ipynb)
```

Comment:
These are HTML files, yes?

Reply:
These are notebook files. Like this the links work here in github.

I already have code to strip suffixes (".md") for local links while files are being moved to the site. I'll be sure that it applies to ".ipynb" as well.

Reply:
gotcha 👌